modules/tinymce/src/plugins/table/main/ts/api/TableFormats.ts

```diff
@@ -0,0 +1,39 @@
+/**
```

Comment:
This file isn't technically an API, so this probably should be in a `core` directory instead.